packages/moonstone/TimePicker/TimePickerBase.js

```diff
@@ -110,6 +110,24 @@ const TimePickerBase = kind({
 		 */
 		title: PropTypes.string.isRequired,
 
+		/**
+		 * Sets the hint string read when focusing the joined picker.
+		 *
+		 * @type {String}
+		 * @default 'change a value with up down button'
+		 * @public
+		 */
+		'aria-label': PropTypes.string,
```

Comment:
This needs an `@memberof ...` to keep documented properly.

Reply:
I have renamed the aria-label prop of TimePickerBase to another prop.
Do you have a prop that requires @memerof to be appended?